cmd/opampsupervisor/.goreleaser.yml

```diff
@@ -0,0 +1,140 @@
+      - --label=org.opencontainers.image.licenses=Apache-2.0
+  - goos: linux
+    goarch: ppc64le
+    dockerfile: Dockerfile
+    image_templates:
+      - otel/opentelemetry-collector-opampsupervisor:{{ .Version }}-ppc64le
+      - otel/opentelemetry-collector-opampsupervisor:latest-ppc64le
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:{{ .Version }}-ppc64le
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:latest-ppc64le
+    build_flag_templates:
+      - --pull
+      - --platform=linux/ppc64le
+      - --label=org.opencontainers.image.created={{.Date}}
+      - --label=org.opencontainers.image.name={{.ProjectName}}
+      - --label=org.opencontainers.image.revision={{.FullCommit}}
+      - --label=org.opencontainers.image.version={{.Version}}
+      - --label=org.opencontainers.image.source={{.GitURL}}
+      - --label=org.opencontainers.image.licenses=Apache-2.0
+    use: buildx
+docker_manifests:
+  - name_template: otel/opentelemetry-collector-opampsupervisor:{{ .Version }}
+    image_templates:
+      - otel/opentelemetry-collector-opampsupervisor:{{ .Version }}-amd64
+      - otel/opentelemetry-collector-opampsupervisor:{{ .Version }}-arm64
+      - otel/opentelemetry-collector-opampsupervisor:{{ .Version }}-ppc64le
+  - name_template: otel/opentelemetry-collector-opampsupervisor:latest
+    image_templates:
+      - otel/opentelemetry-collector-opampsupervisor:latest-amd64
+      - otel/opentelemetry-collector-opampsupervisor:latest-arm64
+      - otel/opentelemetry-collector-opampsupervisor:latest-ppc64le
+  - name_template: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:{{ .Version }}
+    image_templates:
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:{{ .Version }}-amd64
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:{{ .Version }}-arm64
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:{{ .Version }}-ppc64le
+  - name_template: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:latest
+    image_templates:
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:latest-amd64
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:latest-arm64
+      - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-opampsupervisor:latest-ppc64le
+release:
+  github:
+    owner: open-telemetry
+    name: opentelemetry-collector-releases
+  header: |
+    ### Images and binaries for collector distributions here: https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/{{ .Tag }}
```

Comment:
don't know if this header really makes sense here

Reply:
i kept it here as it's also in the OCB workflow. I don't have a strong preference either way

Reply:
replaced it with a header about the OpAMP supervisor instead.